pkg/labels/selector.go

```diff
@@ -76,109 +76,35 @@ func NewSelector() Selector {
 	return internalSelector(nil)
 }
 
-type internalSelector []Requirement
```

Comment:
in `pkg/labels/selectors.go` as far as I understand `Operator` is not used anymore and the `cont Operator` (the tokens) can be removed (since now come from `pkg/utils/selector`)